avocado/core/job.py

```diff
@@ -434,14 +436,14 @@ def _resolver_check_missing_references(references, resolutions):
         for reference in references:
             results = [res.result for res in resolutions if
                        res.reference == reference]
-            if resolver.ReferenceResolutionResult.SUCCESS not in results:
+            if ReferenceResolutionResult.SUCCESS not in results:
                 missing.append(reference)
         if missing:
             msg = "Could not resolve references: %s" % ",".join(missing)
             raise exceptions.JobTestSuiteReferenceResolutionError(msg)
 
     def _make_test_suite_resolver(self, references, ignore_missing):
-        resolutions = resolver.resolve(references)
+        resolutions = resolve(references)
```

Comment:
This is not related to the commit message, or am I missing something?

Reply:
@clebergnu because of the changes in the loop and how I replaced the `if resolver.ReferenceResolutionResult.SUCCESS` to fit on 80 columns I decided to rewrite the import. Because of that I changed the only other place that uses `resolver`, here.

Makes sense?

Reply:
> @clebergnu because of the changes in the loop and how I replaced the `if resolver.ReferenceResolutionResult.SUCCESS` to fit on 80 columns I decided to rewrite the import. Because of that I changed the only other place that uses `resolver`, here.
> 
> Makes sense?

That was my point... I was missing the other motivation ("fit 80 columns").  So maybe just a note on the commit message to explain the why of this change ("while at it, fit to 80 columns").